providers/implementations/signature/ecdsa_sig.c

```diff
@@ -764,9 +764,6 @@ static int ecdsa_set_ctx_params(void *vctx, const OSSL_PARAM params[])
     if ((ret = ecdsa_common_set_ctx_params(ctx, params)) <= 0)
         return ret;
 
-    if (params == NULL)
```

Comment:
Similarly `ecdsa_common_set_ctx_params()` returns `1` for empty `params`, so we don't `return` at line 765